e2e/cypress/integration/api/iam/01_projects_api.spec.ts

```diff
@@ -1,157 +1,160 @@
-      const admin = JSON.parse(<string>localStorage.getItem('chef-automate-user'));
+      adminIdToken = JSON.parse(<string>localStorage.getItem('chef-automate-user')).id_token;
 
-    // Cypress recommends state cleanup in the before block to ensure
-    // it gets run every time:
-    // tslint:disable-next-line:max-line-length
-    // https://docs.cypress.io/guides/references/best-practices.html#Using-after-or-afterEach-hooks
-    cleanupTestProjects(admin.id_token);
+      // Cypress recommends state cleanup in the before block to ensure
+      // it gets run every time:
+      // tslint:disable-next-line:max-line-length
+      // https://docs.cypress.io/guides/references/best-practices.html#Using-after-or-afterEach-hooks
+      cleanupTestProjects(adminIdToken);
+      cy.cleanupProjectsByIDPrefix(adminIdToken, cypressPrefix);
 
-    cy.request({
-      auth: { bearer: admin.id_token },
-      method: 'POST',
-      url: 'api/v0/ingest/events/chef/node-multiple-deletes',
-      body: {
-        node_ids: [
-          'f6a5c33f-bef5-433b-815e-a8f6e69e6b1b',
-          '82760210-4686-497e-b039-efca78dee64b',
-          '9c139ad0-89a5-44bc-942c-d7f248b155ba',
-          '6453a764-2415-4934-8cee-2a008834a74a'
-        ]
-      },
-      failOnStatusCode: false
-    });
+      cy.request({
+        auth: { bearer: adminIdToken },
+        method: 'POST',
+        url: 'api/v0/ingest/events/chef/node-multiple-deletes',
+        body: {
+          node_ids: [
+            'f6a5c33f-bef5-433b-815e-a8f6e69e6b1b',
+            '82760210-4686-497e-b039-efca78dee64b',
+            '9c139ad0-89a5-44bc-942c-d7f248b155ba',
+            '6453a764-2415-4934-8cee-2a008834a74a'
+          ]
+        },
+        failOnStatusCode: false
+      });
 
-    // use our Admin user's ID token to generate an admin-level API token
-    // for use in the tests as Cypress.env('adminTokenValue')
-    cy.generateAdminToken(admin.id_token);
+      // use our Admin user's ID token to generate an admin-level API token
+      // for use in the tests as Cypress.env('adminTokenValue')
+      cy.generateAdminToken(adminIdToken);
 
-    // create projects or confirm they already exist
-    for (const project of [avengersProject, xmenProject]) {
+      for (const project of [avengersProject, xmenProject]) {
+        cy.request({
+          auth: { bearer: adminIdToken },
+          method: 'POST',
+          url: '/apis/iam/v2beta/projects',
+          body: project
+        });
+      }
+
+      let totalNodes = 0;
       cy.request({
-        auth: { bearer: admin.id_token },
-        method: 'POST',
-        url: '/apis/iam/v2beta/projects',
-        failOnStatusCode: false,
-        body: project
+        auth: { bearer: adminIdToken},
+        headers: {
+          projects: '(unassigned)'
+        },
+        method: 'GET',
+        url: '/api/v0/cfgmgmt/nodes?pagination.size=10'
       }).then((response) => {
-        expect([200, 409]).to.include(response.status);
```

Comment:
Actual change 3: removed `failOnStatusCode: false`. If we get a non-2xx code back from creating the projects we should fail because 1) all cypress projects should be cleaned up at this point and 2) we are using timestamps in the IDs so we should never conflict anyway. If we conflict there is a problem that we shouldn't ignore.